events/code_review_events/workflow.py

```diff
@@ -376,7 +373,7 @@ def __init__(self, cache_root):
             else:
                 self.community_monitoring = None
 
-            self.bugbug_utils = BugbugUtils()
+            self.bugbug_utils = BugbugUtils(self.workflow.api)
```

Comment:
It would be cleaner to load the reviewers in the workflow, so you don't share the Phabricator API to the bugbug utils

Reply:
I did this so that all bugbug-related stuff is encapsulated in BugbugUtils.

Reply:
So if/when we move it to another project (by having another "events", like https://github.com/mozilla/bugbug/pull/1098), it's easier to move.

Reply:
As discussed on IRC, let's ship it that way until we plan a real move out of Code review infra.
When moving, bugbugutils should extend phabricatorActions like CodeReview class